spec/spec_helper.rb

```diff
@@ -118,7 +119,8 @@
   { provider: :deepseek, model: 'deepseek-chat' },
   { provider: :openai, model: 'gpt-4.1-nano' },
   { provider: :openrouter, model: 'anthropic/claude-3.5-haiku' },
-  { provider: :ollama, model: 'mistral-small3.1' }
+  { provider: :ollama, model: 'mistral-small3.1' },
+  { provider: :perplexity, model: 'gpt-4.1-nano', }
```

Comment:
@joaoGabriel55 I think if you remove perplexity from this array it shouldn't be considered a CHAT_MODEL and wont case the spec failures.

The errors are coming becasue the model doesn't support function calling and right now, the tests are assuming everything does.

Reply:
@adenta no. It _is_ a chat model so it stays in there. Plenty of models and providers have quirks that we already skip. Skipping means that when they fix their quirks we simply remove the skip statement. On top of that if you now have this specific model out of the CHAT_MODELS array, you need to duplicate all the tests only for Perplexity.